### PR TITLE
New pto sim lib

### PR DESCRIPTION
**Pull request by jleonqu**
New PR to solve some conflicts in the previous PR #743

### CODE REVIEW

**Comment by kmruehl** [comment]
@jleonqu there are merge conflicts with this PR now that PR #803 has been merged, and it looks like there are files revised here that you didn't mean to include. Let's go through this together during the WEC-Sim working meeting tomorrow.

**Comment by kmruehl** [comment]
@jleonqu looks great, thanks for resolving the merge conflicts with dev! @akeeste thanks for your help!

File: docs/user/code_structure.rst [hunk 1]
```diff
@@ -736,6 +736,83 @@ restrictions on the number of MooringMatrix blocks.
    :width: 400 pt
    :align: center
 
+.. _user-code-structure-cable-class:
+
```
Comment (kmruehl) [inline docs/user/code_structure.rst:740]: This should be `.. _user-code-structure-ptosim-class:`

File: source/objects/ptoSimClass.m [hunk 2]
```diff
@@ -20,129 +20,115 @@
...
+            'radius'                          ,'NOT DEFINED',...            % Rotary generator radius
+            'tau_p'                           ,'NOT DEFINED',...            % Magnet pole pitch
+            'theta_d_0'                       ,'NOT DEFINED',...            % Initial theta value
+            'Bfric'                           ,'NOT DEFINED',...            % Friction coeeficient
+            'lambda_sd_0'                     ,'NOT DEFINED',...            % Stator d-axis flow linkage
+            'Rs'                              ,'NOT DEFINED',...            % Winding resistance
+            'lambda_fd'                       ,'NOT DEFINED',...            % Flux linkage of the stator d winding due to flux produced by the rotor magnets [Wb-turns]
+            'Ls'                              ,'NOT DEFINED',...            % Winding inductance
+            'lambda_sq_0'                     ,'NOT DEFINED',...            % Stator q-axis flow linkage
+            'Rload'                           ,'NOT DEFINED')               % External load
```
Comment (kmruehl) [inline source/objects/ptoSimClass.m:105]: These properties should be camelCase

**Review by kmruehl — CHANGES_REQUESTED** [review]
@jleonqu this is looking good and I think we should prioritize merging it and opening another PR for further revisions. 
These revisions should be made prior to merge:

- [x] Please delete this file: `source/lib/PTO-Sim/BlockFigures/PTOSim_Lib.slx`
- [x] This file should not be empty, please revert: `source/objects/ptoClass.m`

**Comment by jleonqu** [comment]
@kmruehl I am not sure about the second comment "This file should not be empty, please revert: source/objects/ptoClass.m", this is the file I currently have in the PR: [ptoClass.m](https://github.com/jleonqu/WEC-Sim/blob/NewPTOSimLib/source/objects/ptoClass.m)

**Comment by kmruehl** [comment]
That's strange, this is what the review is showing on my end:
image

Nothing should have been updated in the `ptoClass`, correct?

**Comment by kmruehl** [comment]
@jleonqu I'd like to merge this into dev but now there are some merge conflicts, can you resolve them?

**Comment by jleonqu** [comment]
@kmruehl I have resolved the conflicts

**Review by kmruehl — APPROVED** [review]
Thanks @jleonqu!

**Pull request merged**